test_scripts/Polices/appID_Management/163_ATF_P_TC_HMI_Status_Value_Of_AppId_In_PT_Is_Null.lua

```diff
@@ -0,0 +1,91 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [HMI Status]: The value of <appID> in LocalPT is null
+--
+-- Description:
+-- SDL should only allow an HMILevel of NONE to the app
+-- in case PolicyTable has "<appID>": "null" in the Local PolicyTable
+--
+-- Preconditions:
+-- 1. appID="123_xyz" is registered to SDL yet
+-- 2. appID="123_xyz" has null policies
+-- Steps:
+-- 1. Activate app
+-- 2. Verify status of activation
+-- 3. Verify app hmi level
+--
+-- Expected result:
+-- 1. status = "REJECTED"
+-- 2. hmiLevel = "NONE"
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local mobileSession = require("mobile_session")
+local testCasesForPolicyAppIdManagament = require("user_modules/shared_testcases/testCasesForPolicyAppIdManagament")
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local commonSteps = require("user_modules/shared_testcases/commonSteps")
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ General Precondition before ATF start ]]
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require("connecttest")
+require("user_modules/AppTypes")
+
+local HMIAppID
```

Comment:
@dboltovskyi please initialize this variable with some value to figure out in logs, that it is wrong if registration was not succeed